src/test.js

```diff
@@ -154,9 +154,9 @@ Test.prototype = {
 		checkPollution();
 	},
 
-	queueHook: function( hook, hookName, hookOwner ) {
-		var promise,
-			test = this;
+	queueHook( hook, hookName, hookOwner ) {
+		const test = this;
```

Comment:
Good suggestion. I can do that, but will need to save it to a variable so that we get a name in the stack trace for better debuggability.